src/helpers.ts

```diff
@@ -56,29 +56,32 @@ function hasNodeExecutable(rootPath: string, executable: string): boolean {
 }
 
 /**
- *  Handles getting the jest runner, handling the OS and project specific work too
+ * Handles getting the jest runner, handling the OS and project specific work too
  *
  * @returns {string}
  */
-export function pathToJest(pluginSettings: IPluginSettings) {
-  if (pluginSettings.pathToJest) {
-    if (isBootstrappedWithCRA(pluginSettings.rootPath)) {
-      return 'npm test --'
-    }
-    return normalize(pluginSettings.pathToJest)
+export function pathToJest({ pathToJest, rootPath }: IPluginSettings) {
+  if (isNotDefaultPathToJest(pathToJest)) {
+    return normalize(pathToJest)
   }
```

Comment:
Moved as suggested @stephtr. ☝️ Thanks.

Reply:
[Your argument](https://github.com/jest-community/vscode-jest/pull/332#discussion_r190103335) is good, I just believe that `if (pathToJest)` ("if pathToJest is set") would be (even for outsiders) more easily readable and understandable than `if (isNotDefaultPathToJest(pathToJest))`.

Reply:
I'll change it to `if (hasUserSetPathToJest(pathToJest))` and call it a day.